rotkehlchen/exchanges/independentreserve.py

```diff
@@ -206,23 +204,23 @@ def _asset_movement_from_independentreserve(raw_tx: dict) -> AssetMovement | Non
     if comment is not None and comment.startswith('Withdrawing to'):
         address = comment.rsplit()[-1]
 
-    raw_amount = raw_tx.get('Credit') if movement_type == AssetMovementCategory.DEPOSIT else raw_tx.get('Debit')  # noqa: E501
+    raw_amount = raw_tx.get('Credit') if movement_type == HistoryEventType.DEPOSIT else raw_tx.get('Debit')  # noqa: E501
 
     if raw_amount is None:  # skip
         return None   # Can end up being None for some things like this: 'Comment': 'Initial balance after Bitcoin fork'  # noqa: E501
     amount = deserialize_asset_amount(raw_amount)
 
     return AssetMovement(
         location=Location.INDEPENDENTRESERVE,
-        category=movement_type,
-        address=address,
-        transaction_id=transaction_id,
-        timestamp=timestamp,
+        event_type=movement_type,
+        timestamp=ts_sec_to_ms(timestamp),
```

Comment:
you can just do `ts_sec_to_ms(deserialize_timestamp_from_date(
        date=raw_tx['CreatedTimestampUtc'],
        formatstr='iso8601',
        location='IndependentReserve',
    ))`